### PR TITLE
Fixed #26729 -- Allowed overriding the label of a tabular inline form field

### PR DESCRIPTION
**Pull request by czpython**
https://code.djangoproject.com/ticket/26729

### CODE REVIEW

File: django/contrib/admin/helpers.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def fields(self):
                     'help_text': help_text_for_field(field_name, self.opts.model),
                 }
             else:
-                form_field = self.formset.form.base_fields[field_name]
+                form_field = self.formset.empty_form.fields[field_name]
```
Comment (charettes) [inline django/contrib/admin/helpers.py]: Neat trick.

**Comment by timgraham** [comment]
Could you try to reuse an existing set of models? We try to avoid new models unless absolutely necessary since they slow the test suite.

Also, if it's possible to write the assertion by accessing the appropriate object in `response.context`, that also makes it easier to debug the test in the event of a failure.

**Comment by czpython** [comment]
@timgraham 
You got it. Regarding your second comment, I'm not sure I follow.. what do you mean by "accessing the appropriate object"? The code in question is [here](https://github.com/django/django/blob/master/django/contrib/admin/templates/admin/edit_inline/tabular.html#L13)

**Comment by timgraham** [comment]
Something like `response.context['inline_admin_formset'].fields` if that works.

**Comment by czpython** [comment]
@timgraham 
Sure however it looks a bit implementation specific

```python
response = self.client.get(reverse('admin:admin_inlines_anotherparentmodel_add'))
field = list(response.context['inline_admin_formset'].fields())[0]
self.assertEqual(field['label'], 'new label')
```

**Comment by timgraham** [comment]
I think it's an improvement since that gives some idea of where to look in the event of a test breakage. With `assertContains`, you really don't have much of an idea. I guess if you want to put the `assertContains` after those other lines, it would be okay too.

**Comment by czpython** [comment]
@timgraham Thanks for review. I've updated the pr to reuse model and do more explicit test.

**Comment by timgraham** [comment]
merged in 9c2d5a8d333277cc1b482a9d05f174cf4d09f24c, thanks!

I removed the ticket references. For future reference, we typically include them for obscure issues where the additional context of the ticket is helpful in understanding the issue.

**Pull request closed without merging**